unattended_installer/install_functions/installCommon.sh

```diff
@@ -293,7 +305,11 @@ function installCommon_installPrerequisites() {
         not_installed=()
 
         for dep in "${dependencies[@]}"; do
-            if ! apt list --installed 2>/dev/null | grep -q "${dep}"; then
+            if [ "${dep}" == "curl" ]; then
+                if ! apt list --installed 2>/dev/null | grep -q -E ^"${dep}"; then
```

Comment:
Use this for all of the dependencies not only curl.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/2063/commits/b7ebab8a5f1ad6a10653c3e40e742a81d2019964.